src/main/controllers/helpers/EmploymentAnswersHelper.ts

```diff
@@ -1,7 +1,28 @@
-import { CaseWithId, StillWorking, YesOrNo, YesOrNoOrNotSure } from '../../definitions/case';
-import { InterceptPaths, PageUrls } from '../../definitions/constants';
-import { TypesOfClaim } from '../../definitions/definition';
-import { AnyRecord } from '../../definitions/util-types';
+import {CaseWithId, PayInterval, StillWorking, WeeksOrMonths, YesOrNo, YesOrNoOrNotSure} from '../../definitions/case';
+import {InterceptPaths, PageUrls} from '../../definitions/constants';
+import {TypesOfClaim} from '../../definitions/definition';
+import {AnyRecord} from '../../definitions/util-types';
+
+let getTranslationsForStillWorkingEnum = function (userCase: CaseWithId, translations: AnyRecord) {
+  var translation = translations.employmentDetails.working;
```

Comment:
Not sure 100% but maybe 'let' instead of 'var' to make it 'method visible'?

Reply:
done